packages/font/src/google/fetch-css-from-google-fonts.ts

```diff
@@ -1,24 +1,10 @@
 // @ts-ignore
 import fetch from 'next/dist/compiled/node-fetch'
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
[`import/no-extraneous-dependencies`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md) is now useless because of [ncc](https://www.npmjs.com/package/@vercel/ncc), right?

introduced by https://github.com/vercel/next.js/pull/13482
disable directives: https://github.com/search?q=repo%3Avercel%2Fnext.js+%22import%2Fno-extraneous-dependencies%22+%22eslint-disable%22&type=code

current configuration:

https://github.com/vercel/next.js/blob/9b2f29ebaf5add67cec0cac7296f9e64567b3fb4/.eslintrc.json#L163-L176